plugins/modules/monitoring/datadog/datadog_event.py

```diff
@@ -113,6 +113,7 @@ def main():
         argument_spec=dict(
             api_key=dict(required=True, no_log=True),
             app_key=dict(required=True, no_log=True),
+            api_host=dict(required=False, default='https://api.datadoghq.com'),
```

Comment:
Instead of hard-coding the value here, why not simply let the library choose the default? (It could also use env variables for that, for example, or some config file.)

It's probably best to leave `default` away here, and only add `api_host` to the `options` dict if `module.params['api_host'] is not None`.

Reply:
You're absolutely correct! I changed the code accordingly!